packages/web-react/scripts/helpers.ts

```diff
@@ -1,6 +1,6 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-import { glob } from 'glob';
 import * as path from 'path';
```

Comment:
```js
import * as path from 'path';
import * as recast from 'recast';
import * as parser from 'recast/parsers/babel';
import { glob } from 'glob';
```

I think this should be sorted this way. E.g. namespaced import precede the named import.